sympy/stats/frv_types.py

```diff
@@ -440,6 +427,18 @@ def Binomial(name, n, p, succ=1, fail=0):
     >>> density(X).dict.subs(n, 4).doit()
     {0: 1/16, 1: 1/4, 2: 3/8, 3: 1/4, 4: 1/16}
 
+    Parameters
+    ==========
+
+    n : Positive Integer
+      Represents number of trials
```

Comment:
The word "Represents" seems unnecessary

Reply:
Agreed. I went ahead and removed all such uses of "Represents" from parameter descriptions throughout the stats module, though there still remains similar uses of other words; I will address them separately.